srcpkgs/python3-Flask-SocketIO/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'python3-Flask-SocketIO'
+pkgname=python3-Flask-SocketIO
+version=5.1.1
+revision=1
+wrksrc=Flask-SocketIO-${version}
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-Sphinx"
+depends="python3  python3-Flask python3-socketio python3-eventlet"
+checkdepends="python3-pytest tox iana-etc ${depends}"
+short_desc="Socket.IO integration for Flask applications"
+maintainer="DragonGhost7 <darkiridiumghost@gmail.com>"
+license="MIT"
+homepage="https://github.com/miguelgrinberg/Flask-SocketIO"
+distfiles="${homepage}/archive/refs/tags/v${version}.tar.gz"
+checksum=b374a0918d61c0cb0ccad50324e41b6c11468026dba776d212ad775777ad4fe5
+
+pre_check() {
+#replace py39 with py310 in tox config file since void switched to py3.10
+	vsed -i '2 s/py39/py310/' tox.ini
+}
+
+do_check() {
+	tox
+}
```

Comment:
We don't want to run tests with tox, because tox creates an environment with its own prefered dependency versions, so it doesn't actually check anything.

Reply:
yeah its just that source code was inside src/ subdirectory so couldnt be imported by test script, but i solved that, no more tox.